libs/sam-formly-sample/src/lib/feature/filter-wrapper/filter-wrapper.component.ts

```diff
@@ -271,6 +274,91 @@ export class FilterWrapperComponent implements OnInit {
           },
         },
       ]
+    },
+    {
+      key: 'conditionalFilters',

```

Comment:
We discussed creating a separate page for conditional field usage and removing it from filters.

Reply:
this is still in progress and changes will be visible in next commit.